Formula/aircrack-ng.rb

```diff
@@ -18,25 +17,26 @@ class AircrackNg < Formula
   depends_on "pkg-config" => :build
   depends_on "sqlite"
   depends_on "openssl"
+  depends_on "make"
+  depends_on "pcre"
 
   # Remove root requirement from OUI update script. See:
   # https://github.com/Homebrew/homebrew/pull/12755
   patch :DATA
```

Comment:
There's a patch here that is needed for aircrack-ng on OS X and it would be good if it was sent to its developers.